app/models/service_template_ansible_playbook.rb

```diff
@@ -1,6 +1,7 @@
 class ServiceTemplateAnsiblePlaybook < ServiceTemplateGeneric
   before_destroy :check_retirement_potential, :prepend => true
   around_destroy :around_destroy_callback, :prepend => true
+  after_rollback :rollback_job_templates
```

Comment:
The callback is going to run for create, update, or destroy rollbacks.  I think we only want this for create.